### ISSUE TITLE
What IPs or Domains are needed?

### ISSUE THREAD

**Issue by markd89** [ram-on/SkyTube#551]
Hi,
I have blocked a lot of IPs, I realize that the video stream is going to come from one or more Google IPs and that I probably need to allow 443  to these sites.

What's the minimal IPs or Domains needed for the app to serve videos?

Thanks!
Mark

**Comment by markd89** [comment]
Inside the code, don't you connect to a particular domain to download a stream?

**Comment by gzsombor** [comment]
Yes, to www.youtube.com, which returns an address, like "r1---sn-c0q7lnsl.googlevideo.com"...
